### ISSUE TITLE
[Bug]: Crash on server start related with golden_oak_tree

### ISSUE THREAD

**Issue by CopperGiraffe** [Zepalesque/Aether-Redux-Issue-Tracker#88]
### Minecraft Version

1.20.1

### Mod Version

2.0.7

### Forge/Neoforge Version

47.2.1

### What happened?

Impossible to start the server using the mod, the embed data_pack is causing an issue (see log)

I’m also using other Aether mods (ancient aether, the lost content…), but if I remove Aether Redux, the problem goes away

### What was the expected result?

I expect the server to start

### Steps to Reproduce

Just try to launch the server

### Crash Report (if applicable)
```
20Mar2024 13:32:07.978] [main/ERROR] [com.structureessentials.StructureEssentials/]: Unbound values in registry ResourceKey[minecraft:root / minecraft:worldgen/configured_feature]: [aether:golden_oak_tree]
[20Mar2024 13:32:07.980] [main/ERROR] [net.minecraft.resources.RegistryDataLoader/]: Registry loading errors:
> Errors in registry minecraft:worldgen/configured_feature:
>> Errors in element aether:golden_oak_tree:
java.lang.IllegalStateException: Failed to parse aether:worldgen/configured_feature/golden_oak_tree.json from pack builtin/aether_redux_compat
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader.m_255048_(RegistryDataLoader.java:147)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader$RegistryData.m_254786_(RegistryDataLoader.java:161)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader.m_254785_(RegistryDataLoader.java:63)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader.m_247207_(RegistryDataLoader.java:62)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.WorldLoader.m_246152_(WorldLoader.java:54)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.WorldLoader.m_245736_(WorldLoader.java:58)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.WorldLoader.m_214362_(WorldLoader.java:31)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.Main.lambda$main$2(Main.java:167)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.Util.m_214652_(Util.java:777)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.Util.m_214679_(Util.java:772)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.Main.main(Main.java:166)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.1/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
	at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.1/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
	at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.1/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
Caused by: java.lang.RuntimeException: Unknown registry key in ResourceKey[minecraft:root / minecraft:worldgen/tree_decorator_type]: aether_redux:patch_tree_decorator
	at MC-BOOTSTRAP/datafixerupper@6.0.8/com.mojang.serialization.DataResult.lambda$getOrThrow$5(DataResult.java:101)
	at MC-BOOTSTRAP/datafixerupper@6.0.8/com.mojang.datafixers.util.Either$Right.map(Either.java:99)
	at MC-BOOTSTRAP/datafixerupper@6.0.8/com.mojang.serialization.DataResult.getOrThrow(DataResult.java:93)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader.m_255048_(RegistryDataLoader.java:143)
	... 26 more

[20Mar2024 13:32:07.980] [main/WARN] [net.minecraft.server.Main/]: Failed to load datapacks, can't proceed with server load. You can either fix your datapacks or reset to vanilla with --safeMode
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Failed to load registries due to above errors
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at net.minecraft.server.Main.main(Main.java:195) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.2.1.jar%2369!/:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103) ~[fmlloader-1.20.1-47.2.1.jar%2369!/:?]
	at net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27) ~[fmlloader-1.20.1-47.2.1.jar%2369!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?]
Caused by: java.lang.IllegalStateException: Failed to load registries due to above errors
	at net.minecraft.resources.RegistryDataLoader.m_247207_(RegistryDataLoader.java:77) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.server.WorldLoader.m_246152_(WorldLoader.java:54) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.server.WorldLoader.m_245736_(WorldLoader.java:58) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.server.WorldLoader.m_214362_(WorldLoader.java:31) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.server.Main.lambda$main$2(Main.java:167) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.Util.m_214652_(Util.java:777) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.Util.m_214679_(Util.java:772) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	at net.minecraft.server.Main.main(Main.java:166) ~[server-1.20.1-20230612.114412-srg.jar%231101!/:?]
	... 15 more
```

### Additional Context/Information

Thanks in advance :)

### Make sure to confirm these as well before submitting the report:

- [X] The issue persists on the latest version of the mod.
- [X] All version numbers are properly recorded, no instances of ‘latest’
- [X] All details are reported clearly and concisely
- [X] This issue is not a duplicate of another issue
- [X] If the issue is a modpack-related issue, it has been reported to the modpack devs as well

**Comment by Zepalesque** [comment]
ah yeah this is an ancient aether compat issue iirc, but I can fix it in 2.0.7c

**Comment by CopperGiraffe** [comment]
Glad to hear, thanks for your work!

**Comment by Zepalesque** [comment]
hmm actually, what ancient aether version are you using?

**Comment by CopperGiraffe** [comment]
Not the last one, I had issue with bonemeal (caused a crash) so 0.8.7 or 0.8.6

**Comment by Zepalesque** [comment]
should be fixed in 2.0.7c
